lib/honeybadger/instrumentation.rb

```diff
@@ -22,14 +22,14 @@ def self.monotonic_timer
       [((finish_time - start_time) * 1000).round(2), result]
     end
 
-    def self.histogram(name, attributes: {}, duration: nil)
+    def self.measure(name, attributes: {}, duration: nil)
       if block_given?
         duration = monotonic_timer{ yield }[0]
       end
 
       raise 'No duration found' if duration.nil?
 
-      attributes.merge!(metric_type: "histogram", metric_name: name)
+      attributes.merge!(metric_type: "measure", metric_name: name)
```

Comment:
Would it make sense to change `metric_type` to "duration"?

Reply:
I'm thinking of renaming this method to `timer` as it's more descriptive than measure. That way the method can match metric type, and the metric type makes more sense as a "timer".

Reply:
Since `gauge` and `increment_counter` are verbs, I'd suggest making this method a verb, too, like `time`.